modules/core/shared/src/test/scala/scaladex/core/model/ArtifactTests.scala

```diff
@@ -141,13 +141,70 @@ class ArtifactTests extends AnyFunSpec with Matchers {
       obtained should contain(expected)
     }
   }
+
+  describe("scastieURL") {
+    it("should return None if artifact is SbtPlugin") {
+      createArtifact(
+        groupId = "com.typesafe.sbt",
+        artifactId = "sbt-native-packager_2.10_0.13",
+        version = "1.2.2",
+        binaryVersion = BinaryVersion(SbtPlugin.`0.13`, Scala.`2.10`)
+      ).scastieURL should be(None)
+    }
+
+    it("should return None if artifact is ScalaNative") {
+      createArtifact(
+        groupId = "org.typelevel",
+        artifactId = "cats-core",
+        version = "2.6.1",
+        binaryVersion = BinaryVersion(ScalaNative.`0.4`, Scala.`2.13`)
```

Comment:
Here as well, this is inconsistent. If the binary version is `BinaryVersion(ScalaNative.`0.4`, Scala.`2.13`)` then the `artifactId` should be `cats-core_native0.4_2.13`